data/bioscience-overview/index.html

```diff
@@ -17,66 +16,66 @@
-<td style="text-align: left;">map data</td>
-<td style="text-align: left;">DNR, Utah Division of Wildlife Resources (DWR)</td>
-<td style="text-align: left;">Habitat areas for game species of Utah and generalized occurrence data for habitat areas of protected species.</td>
-</tr>
-<tr>
-<td style="text-align: left;">Wildfire Perimeters</td>
-<td style="text-align: left;">map data</td>
-<td style="text-align: left;">AGRC, SITLA, DNR, Utah Div. Emergency Management</td>
-<td style="text-align: left;">Perimeters of wildfires in Utah.</td>
-</tr>
-</tbody>
+  <thead>
+    <tr>
+      <th width="25%" scope="col">Product Name</th>
+      <th scope="col">Type</th>
+      <th scope="col">Steward(s)</th>
+      <th width="40%" scope="col">Description</th>
+    </tr>
+  </thead>
+  <tfoot>
+    <tr>
+      <td colspan="5"><sub><em>click a product name for more information and download links</em></sub></td>
+    </tr>
+  </tfoot>
+  <tbody>
+    <tr>
+      <td><a href="https://drive.google.com/drive/folders/0ByStJjVZ7c7mZmJtZXNPN1l3ZFE">Beaver Restoration Assessment</a></td>
+      <td>map data</td>
+      <td>US Fish and Wildlife Service</td>
+      <td>The Utah statewide Beaver Restoration Assessment Tool (BRAT) is a decision support and planning tool for beaver management. The backbone to BRAT is a capacity model developed to assess the upper limits of riverscapes to support beaver dam-building
+        activities. Both existing and potential capacity were estimated with readily available spatial datasets to evaluate five key lines of evidence: 1) a perennial water source, 2) availability of dam building materials, 3) ability to build a dam
+        at baseflow, 4) likelihood of dams to withstand a typical flood, and 5) likelihood that stream gradient would limit or completely eliminate dam building by beaver. Fuzzy inference systems were used to combine these lines of evidence while accounting
+        for uncertainty.</td>
+    </tr>
+    <tr>
+      <td><a href="https://drive.google.com/drive/folders/0ByStJjVZ7c7mNEFsbWhudnI3SHc">Dominant Vegetation</a></td>
+      <td>map data</td>
+      <td>Utah Division of Wildlife Resources (DWR)</td>
+      <td>Statewide distribution of dominant vegetation species in Utah. The data were derived from 1:24,000 scale orthophoto quadrangles, 1:40,000 scale aerial photos and field mapping by the Utah Division of Wildlife Resources (approximate date Jan. 2001).</td>
+    </tr>
+    <tr>
+      <td>GAP Landcover Classification (Raster)</td>
+      <td>map data and map services</td>
+      <td>USGS GAP Analysis Program, USU RS/GIS Lab</td>
+      <td>Landcover classification for the State of Utah for the purposes of regional terrestrial biodiversity assessment.<br /><a href="http://earth.gis.usu.edu/swgap/landcover_download.html"> Download data files</a><br />
```

Comment:
I actually thought that they were warranted...
with:
![image](https://user-images.githubusercontent.com/1326248/29130410-f75c8a84-7ce6-11e7-8f35-5d60be540365.png)
without:
![image](https://user-images.githubusercontent.com/1326248/29130447-20c643a6-7ce7-11e7-8325-a78c0bca74fc.png)

But you are the website czar. So let me know what you think.